client/app/app.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+let throwcast = angular.module('throwcast', [
```

Comment:
Also since you are just chaining everything in this file. There is no reason to create this variable and assign it.

from:
```
let throwcast = angular.module('throwcast', [
```
to:

```
angular.module('throwcast', [
```